### PR TITLE
Add TugOfWar demo

### PR DESCRIPTION
**Pull request by ChadliaJerad**
This PR adds the Tug of War demo under experiments.
The demo uses Flask and Dash to build the UI. Further details are in the [README.md](https://github.com/lf-lang/playground-lingua-franca/blob/tug-of-war-flask-ui/experiments/C/src/TugOfWar/README.md) file.

### CODE REVIEW

File: experiments/C/src/TugOfWar/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+This example features a browser-based UI. The server is constructed using [Flask](https://flask.palletsprojects.com/en/3.0.x/), a Python web framework, and [Dash](https://dash.plotly.com/) components.
+
+Every second, player forces are generated and their values are send to the server as 
+a post request. The UI's gauges are updated with these values, along with the position of the mark (yellow square). When the mark reaches one of the limits (pink diamond), the label on the top of the page is updated with the result.
+
+## Steps:
+
+ 1. Compile `TugOfWar.lf`. 
+ 2. Launch the UI by runnig the script `start.sh` under `C/src/TugOfWar/Interface`. The script will create a virtual environment and install the requirements listed in `requirements.txt`, if not there already. It then lauches Falsk server, accessible on: [http://127.0.0.1:5004](http://127.0.0.1:5004).
```
Comment (edwardalee) [inline experiments/C/src/TugOfWar/README.md:15]:
```suggestion
 2. Launch the UI by running the script `start.sh` under `C/src/TugOfWar/Interface`. The script will create a virtual environment and install the requirements listed in `requirements.txt`, if not there already. It then lauches Falsk server, accessible on: [http://127.0.0.1:5004](http://127.0.0.1:5004).
```

Comment (edwardalee) [inline experiments/C/src/TugOfWar/README.md:15]:
```suggestion
 2. Launch the UI by running the script `start.sh` in the `C/src/TugOfWar/Interface` directory. The script will create a virtual environment and install the requirements listed in `requirements.txt`, if not there already. It then lauches Flask server, accessible on: [http://127.0.0.1:5004](http://127.0.0.1:5004).
```

File: experiments/C/src/TugOfWar/Interface/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+flask==2.2.0
+dash==2.11.1
+Werkzeug==2.2.0
+pandas
+plotly_express
+dash_bootstrap_components
+dash_daq
```
Comment (edwardalee) [inline experiments/C/src/TugOfWar/Interface/requirements.txt:7]:
```suggestion
dash_daq
setuptools

```

**Review by edwardalee — CHANGES_REQUESTED** [review]
This is a great example!  I've suggested one significant change, which is that requirements.txt needs to include `setuptools` (at least it did for me).  Other suggestions are just typos.  Let's get this merged!

It would be fun to see about realizing the Python side of this using a Python target LF program, but that's another project.

File: experiments/C/src/TugOfWar/README.md (hunk 1 above)
Comment (ChadliaJerad) [reply experiments/C/src/TugOfWar/README.md]: It seems that the suggestions from the previous and current commits have overlapped, causing this one to become outdated. The commit is performed manually in ([28c902e](https://github.com/lf-lang/playground-lingua-franca/pull/92/commits/28c902ecd76c77ee0d6adb4e71f4a967432a7119)).

**Review by edwardalee — APPROVED** [review]
LGTM!

**Pull request merged**